cmd/bundle/generate/utils.go

```diff
@@ -103,8 +119,19 @@ func (n *downloader) markNotebookForDownload(ctx context.Context, notebookPath *
 
 	ext := notebook.GetExtensionByLanguage(info)
 
-	filename := path.Base(*notebookPath) + ext
-	targetPath := filepath.Join(n.sourceDir, filename)
+	if n.basePath == "" {
```

Comment:
Same as above, let's not mutate state if possible.

Reply:
Thanks, fixed